modules/tinymce/src/plugins/table/main/ts/selection/TableSelection.ts

```diff
@@ -5,13 +5,20 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-import { CellOpSelection, Selections } from '@ephox/darwin';
+import { CellOpSelection, Selections, TableSelection } from '@ephox/darwin';
 import { Arr, Optionals } from '@ephox/katamari';
 import { TableLookup } from '@ephox/snooker';
-import { Attribute, Compare, SelectorFind, SugarElement, SugarElements } from '@ephox/sugar';
+import { Attribute, Compare, SelectorFind, SugarElement, SugarElements, SugarNode } from '@ephox/sugar';
+import { ephemera } from 'tinymce/plugins/table/selection/Ephemera';
```

Comment:
Should this be a relative import?

Reply:
Good catch, this should be a relative import.

Reply:
That was probably my VSCode messing up the imports again. Thanks!

Reply:
Fixed